docs/mine/lotus/sector-pledging.md

```diff
@@ -75,3 +75,43 @@ lotus-miner sectors mark-for-upgrade <sector number>
 ```
 
 The sector should become inactive within 24 hours after a new replacement sector has sealed. From that point, the pledged storage can be re-used for new sectors.
+
+## Inspect expiring sectors
+First you can check the sectors that are about to expire. Sectors that will expire within 60 days can be checked by default with the following command. 
+```sh
+lotus-miner sectors check-expire
+```
+If you want to check for sectors that will expire within six months, add the --cutoff option, timing in epoch, here is an example:
+```sh
+lotus-miner sectors check-expire --cutoff 518400
+```
+
+## Extend sectors
+You can extend the lifecycle of a sector with the command.
```

Comment:
Change `.` to `:` when you're showing the user an example in a following codeblock.